ow_lander/src/ow_lander/mixins.py

```diff
@@ -311,7 +311,55 @@ def move_pan_and_tilt(self, pan, tilt):
       rate.sleep()
     raise RuntimeError("Timed out waiting for pan/tilt values to reach goal.")
 
+# NOTE: the following move_pan and move_tilt functions have been
+# dumbly factored out of the previous function.  The FIXME comments
+# above have been omitted but apply.  A more refined refactoring is
+# left to a future iteration.
```

Comment:
I agree with refactoring later since the FIXME comment will require changes to these functions anyways.